pulsar-client-go/pulsar/consumer_test.go

```diff
@@ -98,7 +100,10 @@ func TestConsumer(t *testing.T) {
 		assert.NotNil(t, msg)
 
 		assert.Equal(t, string(msg.Payload()), fmt.Sprintf("hello-%d", i))
-		assert.Equal(t, string(msg.Topic()), "persistent://public/default/my-topic")
+		assert.Equal(t, string(msg.Topic()), "persistent://public/default/" + topic)
```

Comment:
```suggestion
		assert.Equal(t, msg.Topic(), "persistent://public/default/" + topic)
```

The type of msg.Topic itself is string, we don't need to convert here.